mnefun/_mnefun.py

```diff
@@ -108,11 +108,13 @@ def __init__(self, tmin=None, tmax=None, t_adjust=0, bmin=-0.2, bmax=0.0,
         self.n_jobs_resample = n_jobs_resample
         self.filter_length = filter_length
         self.cont_hp = None
+        self.cont_hp_trans = 0.5
         self.cont_lp = 5.
-        self.lp_cut = lp_cut
+        self.cont_lp_trans = None
         self.hp_cut = hp_cut
-        self.lp_trans = lp_trans
+        self.lp_cut = lp_cut
```

Comment:
I got an error when `p.cont_lp_trans` was none, so I made sure it was set to a reasonable value.

Reply:
Ahh thanks, fixed